packages/components/atoms/f-button/src/components/Link.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+    <a
+        :href="$attrs.href"
+        target="_blank">
```

Comment:
I don't think we need to rewrite a default target prop here

Reply:
Do we need `href` passed through as well?

Reply:
👍  I've removed both